### PR TITLE
ONE 10 cards

### PR DESCRIPTION
**Pull request by Simisays**
Nothing too hard here

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Blazing Crescendo
+ManaCost:1 R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ +3 | NumDef$ +1 | SubAbility$ DBExile | SpellDescription$ Target creature gets +3/+1 until end of turn.
+SVar:DBExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. You may play that card this turn.
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ Play | SubAbility$ DBCleanup | ForgetOnMoved$ Exile
+SVar:Play:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play remembered card.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SpellDescription$ Exile the top card of your library. Until the end of your next turn, you may play that card.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt:9]: this line isn't needed?

File: forge-gui/res/cardsfolder/upcoming/exuberant_fuseling.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Exuberant Fuseling
+ManaCost:R
+Types:Creature Phyrexian Goblin Warrior
+PT:0/1
+K:Trample
+S:Mode$ Continuous | Affected$ Card.Self | AffectedZone$ Battlefield | AddPower$ X | Description$ CARDNAME gets +1/+0 for each oil counter on it.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ When CARDNAME enters the battlefield and whenever another creature or artifact you control is put into a graveyard from the battlefield, put an oil counter on Exuberant Fuseling.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/exuberant_fuseling.txt:7]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/exuberant_fuseling.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.Other+YouCtrl,Artifact.Other+YouCtrl | Execute$ TrigCounter | Secondary$ True | TriggerDescription$ When CARDNAME enters the battlefield and whenever another creature or artifact you control is put into a graveyard from the battlefield, put an oil counter on Exuberant Fuseling.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/exuberant_fuseling.txt:8]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Blazing Crescendo
+ManaCost:1 R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ +3 | NumDef$ +1 | SubAbility$ DBExile | SpellDescription$ Target creature gets +3/+1 until end of turn.
+SVar:DBExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. You may play that card this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt:5]: Missing/Wrong Duration

File: forge-gui/res/cardsfolder/upcoming/escaped_experiment.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Escaped Experiment
+ManaCost:1 U
+Types:Artifact Creature Phyrexian Beast
+PT:2/1
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDebuff | TriggerDescription$ Whenever CARDNAME attacks, target creature an opponent controls gets -X/-0 until end of turn, where X is the number of artifacts you control.
+SVar:TrigDebuff:DB$ Pump | | NumAtt$ -X | NumDef$ -0 | ValidTgts$ Creature.OppCtrl
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/escaped_experiment.txt:6]: |  |

File: forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt (hunk 4 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/blazing_crescendo.txt:5]: Still wrong Description

**Review by tool4ever — APPROVED** [review]

**Pull request merged**